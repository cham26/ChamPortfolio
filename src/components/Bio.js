import React from 'react'
import { Container, Row, Col, Button } from 'reactstrap'
import { css } from 'emotion'
import bio from '../assets/bio2-min.jpeg'
import BaseImage from './Base/BaseImage'
import ResumeModal from './ResumeModal'

const override = css`
	.buttonStyle {
		width: 230px;
		font-size: 18px;
		background-color: lightcoral;
		border-color: white;
		font-family: monospace;
	}
	.headingStyle {
		font-family: fantasy;
		font-size: 50px;
	}
	.noteStyle {
		font-family: verdana;
		text-align: justify;
		font-size: 16px;
	}
	.labelColor {
		color: lightcoral;
	}
`

const email = 'charmaine.stepaniuk@gmail.com'

export default function Bio() {
	const onContactMe = () => {
		window.location.href = `mailto:${email}`
	}

	const onResumeView = () => {}

	return (
		<Container fluid id="bio-section" className={override}>
			<Row className="py-5 justify-content-center">
				<Col md={{ size: 5, offset: 1 }}>
					<div>
						<h1 className="display-3 headingStyle">
							Strives for <strong className="labelColor">Quality</strong>
							<br />
							Works <strong className="labelColor">Professionally</strong>
						</h1>
					</div>
					<br />
					<div>
						<p className="noteStyle">
							{'Hi there! I\'m '}
							<strong>
								<span className="labelColor">Cham</span>
							</strong>
							{'	, a nurse by profession turned'}
							<strong>
								<span className="labelColor">Virtual Assistant since 2017. </span>
							</strong>
							My goal is to{' '}
							<strong>
								<span className="labelColor">provide services</span>{' '}
							</strong>
							that delight consumers and business owners. I aim to{' '}
							<strong>
								<span className="labelColor">drive innovation</span>{' '}
							</strong>
							across organizations, expand existing customer sales, brand and product evolution,{' '}
							<strong>
								<span className="labelColor">excellent customer satisfaction </span>
							</strong>
							and social media endorsement. <br />
							<br />I am a work in progress but I know I'll always give my best in everything I do. After all, Franklin
							Roosevelt once said,{' '}
							<strong>
								<span className="labelColor">"A smooth sea never made a skilled sailor".</span>
							</strong>
						</p>
					</div>
					<div className="text-center pt-5">
						<Row>
							<Col>
								<Button onClick={onContactMe} className="buttonStyle">
									Contact me
								</Button>
							</Col>
							<Col>
								<ResumeModal buttonLabel="View Online Resume" cssClasses={['buttonStyle']} />
								{/* <Button onClick={onResumeView} className="buttonStyle">
              View Online Resume
            </Button> */}
							</Col>
						</Row>
					</div>
				</Col>
				<Col md={{ size: 5, offset: 1 }} className="py-2 text-center">
					<BaseImage imageSrc={bio} style={{ maxHeight: '500px' }} cssClasses={['rounded-circle img-fluid']} />
				</Col>
			</Row>
		</Container>
	)
}
