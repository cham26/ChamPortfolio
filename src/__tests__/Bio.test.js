import React from 'react'
import { create } from 'react-test-renderer'
import Bio from '../components/Bio'

test('snapshot', () => {
	// Tests if html changed used for finished components
	// to update it run with -u argmunet
	const c = create(<Bio />)
	expect(c.toJSON()).toMatchSnapshot()
})
