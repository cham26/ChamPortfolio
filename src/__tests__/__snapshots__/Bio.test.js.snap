// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="css-95n8ty container-fluid"
  id="bio-section"
>
  <div
    className="py-5 justify-content-center row"
  >
    <div
      className="col-md-5 offset-md-1"
    >
      <div>
        <h1
          className="display-3 headingStyle"
        >
          Strives for 
          <strong
            className="labelColor"
          >
            Quality
          </strong>
          <br />
          Works 
          <strong
            className="labelColor"
          >
            Professionally
          </strong>
        </h1>
      </div>
      <br />
      <div>
        <p
          className="noteStyle"
        >
          Hi there! I'm
           
          <strong>
            <span
              className="labelColor"
            >
              Cham
            </span>
          </strong>
          , a nurse by profession turned
           
          <strong>
            <span
              className="labelColor"
            >
              Virtual Assistant since 2017.
               
            </span>
          </strong>
          My goal is to
           
          <strong>
            <span
              className="labelColor"
            >
              provide services
            </span>
             
          </strong>
          that delight consumers and business owners. I aim to
           
          <strong>
            <span
              className="labelColor"
            >
              drive innovation
            </span>
             
          </strong>
          across organizations, expand existing customer sales, brand and product evolution,
           
          <strong>
            <span
              className="labelColor"
            >
              excellent customer satisfaction
               
            </span>
          </strong>
          and social media endorsement. 
          <br />
          <br />
          I am a work in progress but I know I'll always give my best in everything I do. After all, Franklin Roosevelt once said,
           
          <strong>
            <span
              className="labelColor"
            >
              "A smooth sea never made a skilled sailor".
            </span>
          </strong>
        </p>
      </div>
      <div
        className="text-center pt-5"
      >
        <div
          className="row"
        >
          <div
            className="col"
          >
            <button
              aria-label={null}
              className="buttonStyle btn btn-secondary"
              onClick={[Function]}
              type="button"
            >
              Contact me
            </button>
          </div>
          <div
            className="col"
          >
            <button
              aria-label={null}
              className="buttonStyle btn btn-secondary"
              onClick={[Function]}
              type="button"
            >
              View Online Resume
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="py-2 text-center col-md-5 offset-md-1"
    >
      <div
        className="css-1yajfje"
      >
        <div
          className="css-1vq1ez6"
        />
        <div
          className="css-ry4pbn"
        />
        <div
          className="css-1y3qqjp"
        />
        <div
          className="css-1sndib6"
        />
        <div
          className="css-qqk109"
        />
      </div>
      <img
        alt="did not load"
        className="rounded-circle img-fluid"
        onError={[Function]}
        onLoad={[Function]}
        src={
          Object {
            "process": [Function],
          }
        }
        style={
          Object {
            "maxHeight": "500px",
          }
        }
      />
    </div>
  </div>
</div>
`;
